CONTRIBUTING.md

```diff
@@ -75,29 +80,90 @@ Note that the FLS ID should be filled according to the FLS paragraph ID for whic
 
 You would then pull `fls_4rhjpdu4zfqj` to place in the FLS ID field.
 
-### 2) Create a Draft with a Member
 
-A member of the Coding Guidelines Subcommittee should get you a first review with some feedback within 14 days of submission. You'll work with one or more members to flesh out the concept and ensure the guideline is well prepared.
+### 2) Guideline Generated as a Comment
+
+A GitHub Action will fire, adding a comment to your newly created issue with
+the contents of the coding guideline prepared written out correctly
+in reStructuredText.
+
+Note that if you later update the body of the coding guideline issue this will
+fire the GitHub Action again and update the original comment with the new
+contents converted to reStructuredText.
+
+### 3) Create a Draft with a Member
+
+Within 14 days of your submission, a member of the Coding Guidelines Subcommittee should give you a first review. You'll work with them (and other members) to flesh out the concept and ensure the guideline is well prepared for a Pull Request.
+
+### 4) Create the PR
+
+> [!NOTE]
+> Here's a list of recommended prerequisites that shall be fulfilled before turning an issue into a PR:
+>
+> * The new rule isn't already covered by another rule or, in case there is already another rule, the rules need to be linked and categorized
```

Comment:
```suggestion
> * The new rule isn't already covered by another rule
>     * OR, in case there is already another rule(s),
>     * The existing rule(s) need(s) to be linked to the new rule,
>     * AND the new rule needs to link to the existing rule(s).
```

Making it easier to read. Since it's a list of requirements, I reckon it's good to have a piece-by-piece list.

What do you think, @PLeVasseur?